packages/renderers/renderer-vue/server.js

```diff
@@ -1,5 +1,5 @@
-import { renderToString } from '@vue/server-renderer';
 import { h, createSSRApp } from 'vue';
+import { renderToString } from 'vue/server-renderer';
```

Comment:
Minor change: I was seeing why the sourcemaps weren’t working in the Vue renderer at all, and it’s because we’re compiling to HTML, which doesn’t really get a good sourcemap. We’ll have to tackle this as a later ticket.

But in digging into it a little, Vue recommends this dependency cleanup so I just did it (this saves us from having to keep 3 package versions in sync)